leaflet/index.d.ts

```diff
@@ -1345,11 +1345,11 @@ declare namespace L {
         imagePath: string;
     }
 
-    export namespace Icon {
-        export const Default: IconDefault;
+    export class Icon {
+        static Default: IconDefault;
     }
```

Comment:
Since `options` are not optional (see next comment), `Icon` needs a constructor that accepts `IconOptions`. I also noticed I made a mistake with this. `Default` is not an instance itself, but a class that extends `Icon`. Could you please change it to something like this:

```
export class Icon {
   constructor(options: IconOptions);
}

export namespace Icon {
    export class Default extends Icon {
        constructor(options?: IconOptions);
    }
}
```